extensions/vscode/src/test/suite/utils/cli-version.test.ts

```diff
@@ -23,15 +23,37 @@ suite("readDartFrogCLIVersion", () => {
     sinon.restore();
   });
 
-  test("returns the version of Dart Frog CLI installed in the user's system", () => {
-    const dartFrogVersionCommandResult = "0.3.7\n";
-    const encoededDartFrogVersionCommandResult = new TextEncoder().encode(
-      dartFrogVersionCommandResult
-    );
-    cpStub.execSync.returns(encoededDartFrogVersionCommandResult);
+  test(
+    "returns the version of Dart Frog CLI installed in the user's system " +
+      "when is last version",
+    () => {
+      const dartFrogVersionCommandResult = "0.3.7\n";
+      const encodedDartFrogVersionCommandResult = new TextEncoder().encode(
+        dartFrogVersionCommandResult
+      );
+      cpStub.execSync.returns(encodedDartFrogVersionCommandResult);
 
-    assert.strictEqual(cliVersion.readDartFrogCLIVersion(), "0.3.7");
-  });
+      assert.strictEqual(cliVersion.readDartFrogCLIVersion(), "0.3.7");
+    }
+  );
+
+  test(
+    "returns the version of Dart Frog CLI installed in the user's system " +
```

Comment:
I think we can group these tests in: "returns the version of Dart Frog CLI installed in the user's system".